### PR TITLE
Tweaks pierced realities

### PR DESCRIPTION
**Pull request by Rukofamicom**
## About The Pull Request

1) Dramatically reduces the lifetime of pierced realities 
2) Changes their in-game name to ??? since they are supposed to be incomprehensible phenomena to non-heretics.

Timeline:
* Invisible to non-heretics for first 15 seconds, just like current iteration
* Fade in to being fully visible over 15 seconds
* Fully visible for 10 seconds (30-40 seconds into its total lifetime)
* Fade to nothingness over 15 seconds
* qdel itself after a grand total of 55 seconds

## Why It's Good For The Game

Heretical players aren't outed almost immediately, enabling a longer period of intrigue and more gradual prep instead of rushing. 

Heretics leave *more than ample* evidence behind in so many ways that the persistence of pierced realities are not a necessary part of counterbalancing them and only serve to worsen the experience of heretic rounds by rapidly accelerating security into a state of bag-searching every dweller on the station. 

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

Not really much to show exactly. Here's proof of the reality fading.

image

</details>

## Changelog
:cl:
tweak: Pierced realities are more incomprehensible to the fragile minds of the average space station worker, and also fade from existence much quicker.
/:cl:

### CODE REVIEW

**Comment by february-ops** [comment]
I like the pierced realities having a name, because it references the cult sim fragments and i like references :)

**Comment by EvilDragonfiend** [comment]
I agree 15 minutes are way too long, but 40 seconds seem to be too short as well. 120 seconds would be enough, I think.

File: code/modules/antagonists/eldritch_cult/eldritch_effects.dm [hunk 1]
```diff
@@ -136,7 +136,7 @@
 		smashes++
 
 /obj/effect/broken_illusion
-	name = "pierced reality"
+	name = "???"
 	icon = 'icons/effects/eldritch.dmi'
```
Comment (PowerfulBacon) [inline code/modules/antagonists/eldritch_cult/eldritch_effects.dm]: Not convinced on the name part of this, ??? makes less sense than some obscure name that doesn't tell you much (plus there are toys that need renaming)

Comment (Rukofamicom) [reply code/modules/antagonists/eldritch_cult/eldritch_effects.dm]: 👍 
I'll drop it then

**Review by PowerfulBacon — APPROVED** [review]

**Pull request merged**